cmd/test/cmd.go

```diff
@@ -121,14 +121,8 @@ func Test(ctx context.Context, ioCtrl *io.Controller, k8sLogger *io.K8sLogger, a
 func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtrl *io.Controller, k8sLogger *io.K8sLogger, tracker *ProxyTracker) (analytics.TestMetadata, error) {
 	fs := afero.NewOsFs()
 
-	// Loads, updates and uses the context from path. If not found, it creates and uses a new context
-	if err := contextCMD.LoadContextFromPath(ctx, options.Namespace, options.K8sContext, options.ManifestPath, contextCMD.Options{Show: true}); err != nil {
-		if err.Error() == fmt.Errorf(oktetoErrors.ErrNotLogged, okteto.GetContext().Name).Error() {
-			return analytics.TestMetadata{}, err
-		}
-		if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.Options{Namespace: options.Namespace}); err != nil {
-			return analytics.TestMetadata{}, err
-		}
+	if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.Options{Namespace: options.Namespace}); err != nil {
```

Comment:
I think we've lost the showing message due to this change

Reply:
thanks for noticing it, I've fixed it here: 30d6709ffa4c1c5cdeb67d12931a364ad06cde19